frontend/src/components/landing/Reviews.module.scss

```diff
@@ -0,0 +1,294 @@
+  padding: 0;
+  background-color: $color-white;
+  color: $color-grey-30;
+  width: $layout-sm;
+  cursor: pointer;
+}
+
+.quotation-icon {
+  display: flex;
+  position: absolute;
+
+  @media screen and #{$mq-md} {
+    left: 0;
+  }
+
+  svg {
+    color: $color-purple-60;
+    height: $layout-sm;
+  }
+}
+
+.reviews-container {
+  display: flex;
+  overflow-x: hidden;
+  flex-direction: row;
+  gap: $spacing-lg;
+  // magic number: set to avoid shifting - no character limit on reviews yet.
+  min-height: calc($layout-2xl * 2);
+  position: relative;
+
+  @media screen and #{$mq-xl} {
+    // magic number: set to avoid shifting - no character limit on reviews yet.
+    min-height: 210px;
+  }
+
+  .review-container {
+    display: flex;
+    align-items: center;
+
+    @media screen and #{$mq-xl} {
+      align-items: flex-start;
+    }
+  }
+
+  .review {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    position: relative; // allows us to animate - scroll
+    width: 100%; // allows us to animate - scroll
+    padding: 0 $spacing-sm;
+
+    // queries set to these static widths to prevent shifting
+    @media screen and #{$mq-sm} {
+      width: $content-xs;
+    }
+    @media screen and #{$mq-md} {
+      width: $content-sm;
+    }
+    @media screen and #{$mq-xl} {
+      width: $content-md;
+    }
+
+    .text {
+      width: 100%;
+
+      ul {
+        padding-left: calc($spacing-sm + $spacing-xs);
+      }
+    }
+  }
+
+  .details {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    font-weight: 400;
+    gap: $spacing-md;
+    padding: $spacing-2xl 0 $spacing-md 0;
+    font-family: $font-stack-base;
+
+    @media screen and #{$mq-xl} {
+      flex-direction: row;
+      align-items: flex-end;
+    }
+
+    .name {
+      @include text-title-3xs;
+      font-family: $font-stack-firefox;
+      font-weight: 600;
+    }
+
+    .source {
+      @include text-body-sm;
+      color: $color-grey-30;
+    }
+
+    .star {
+      color: $color-yellow-40;
+      margin: 0 $spacing-xs 0 ($spacing-sm * -1);
+    }
+  }
+}
+
+// scroll animation
+.scroll-left {
+  animation: scrollLeft 500ms cubic-bezier(0.55, -0.4, 0.45, 1.2);
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+}
+
+.scroll-right {
+  animation: scrollRight 500ms cubic-bezier(0.55, -0.4, 0.45, 1.2);
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+}
+
+// scroll animation keyframes
+@keyframes scrollLeft {
+  0% {
+    left: 0;
+    opacity: 0;
+  }
+  10% {
+    left: -250%;
+  }
+  20% {
+    left: 150%;
+    opacity: 0;
+  }
+  100% {
+    left: 0;
+    opacity: 1;
+  }
+}
+@keyframes scrollRight {
+  0% {
+    left: 0;
+    opacity: 0;
+  }
+  10% {
+    left: 250%;
+  }
+  20% {
+    left: -150%;
+    opacity: 0;
+  }
+  100% {
+    left: 0;
+    opacity: 1;
+  }
+}
+
+// mobile only controls for review scrolling
+.mobile-controls {
+  display: flex;
+  gap: 8px;
+
```

Comment:
It looks like these are overridden by `.hidden-mobile` and `.show-mobile`. (Otherwise, I'd have suggested `$spacing-sm` instead of `8px`.)

```suggestion
```